README.md

```diff
@@ -71,26 +71,19 @@ const tracker = new GbTracker('customer_id', 'area', {
 
 The first party cookie `gbi_visitorId` is set with an empiry time of 1 year that is extended each time the shopper visits again. This is used to anonymously track the shopper.
 
-### 5.0.0 changes
-
-**IMPORTANT**
-
-As of version 5.0.0, the way shopper tracking works has changed. It includes cross-domain support, but tracking only works on domains whose apex domain's TLD is not a second-level domain. Examples:
-
-- `.com` -  You can use version 5.0.0. Shoppers will be tracked. If the cross-domain rules are met (see below), they will be tracked across your domains.
-- `.co.uk` - You cannot use version 5.0.0. If you do, shoppers will not be tracked at all because the cookie-setting code will fail silently. You must wait for a version to be published after 5.0.0. The new version will include instructions for re-enabling shopper tracking.
-
 ### Cross-domain rules
 
-For shoppers to be tracked across your domains, in additional to the rule that the TLD must not be a second-level domain (see above), the domains must share the same apex domain. Examples:
+For shoppers to be tracked across your domains, the domains must share the same apex domain. Examples:
   
 - `a.example.com` and `b.example.com`
 - `a.example.com` and `example.com`
 - `example.com` and `a.example.com`
 
-### Upgrading from versions 3 or 4 and visitor ID re-use
+In order to support this functionality for Second-Level-Domains such as .co.uk, a list of domains has been added to the script logic to account for these cases.
+The list is available here ![Link to the list of domains](https://github.com/groupby/gb-tracker-client/src/slds.ts)
+If a SLD you use is not available in the list, please contact support@groupbyinc.com for assistance.
```

Comment:
`![alt](url)` is for images. Regular links use `[text](url)`. The path can also be relative and will be resolved appropriately.

The text can be edited to be shorter:
```suggestion
Starting with version 5.1.2, many two-level domain suffixes such as `.co.uk` are also supported.
If your domain has a multi-level suffix that is not in the [list of supported domain suffixes](src/slds.ts), please contact support@groupbyinc.com for assistance.
```